docker/types.go

```diff
@@ -108,6 +120,22 @@ type Config struct {
 	Shell           strslice.StrSlice   `json:",omitempty"` // Shell for shell-form of RUN, CMD, ENTRYPOINT
 }
 
+// github.com/docker/distribution/manifest/schema1/config_builder.go
```

Comment:
The github web site's URL isn't going to perfectly mirror the tree layout that you get when cloning the repository, and I've been going for the latter here.